djangochannelsrestframework/observer/model_observer.py

```diff
@@ -41,6 +41,7 @@ def __init__(self, func, model_cls: Type[Model], partition: str = "*", **kwargs)
         self._model_cls = None
         self.model_cls = model_cls  # type: Type[Model]
         self.id = uuid4()
+        self._instance_messages_mapping = defaultdict(list)
```

Comment:
Currently we store observer state on the model insurance not the observer instance.

Maybe we should extend `ModelObserverInstanceState` to store this pending messages?  This way the messages will be stored on the model instances so will be garbage collected.